tests/misc/programs/misc/src/account.rs

```diff
@@ -1,50 +1,63 @@
 use anchor_lang::prelude::*;
 
+macro_rules! size {
```

Comment:
nit: the savings here isn't worth the extra jump when reading the code. (no need to change though)